src/sentry/integrations/slack/utils/users.py

```diff
@@ -74,3 +79,63 @@ def get_slack_data_by_user(
                 slack_data_by_user[user] = slack_info_by_email[email]
                 break
     return slack_data_by_user
+
+
+# USING SDK CLIENT
+
+
+@dataclass
```

Comment:
I recommend always writing `@dataclass(frozen=True)` for a new dataclass that doesn't need to be mutated after creation.